File: Server/Server/Models/View_Models/UserTasksInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Models.View_Models
{
    public class UserTasksInfo
    {
        public int UserId { get; set; }
        public string UserName { get; set; }
        public int CountCompletedTasks { get; set; }
        public int CountActiveTasks { get; set; }
    }
}

File: Client/src/app/models/userTasksInfo.ts
export class UserTasksInfo{

    constructor(userId: number, userName: string, countCompletedTasks: number, countActiveTasks: number){
        this.userId=userId;
        this.userName=userName;
        this.countCompletedTasks=countCompletedTasks;
        this.countActiveTasks=countActiveTasks;
    }

    userId: number;
    userName: string;
    countCompletedTasks: number;
    countActiveTasks: number;
}